service-base/src/main/java/org/eclipse/hono/service/management/tenant/TenantManagementService.java

```diff
@@ -48,7 +48,7 @@
      * @see <a href="https://www.eclipse.org/hono/docs/api/management/#/tenants/createTenant">
      *      Device Registry Management API - Create Tenant</a>
      */
-    void add(Optional<String> tenantId, JsonObject tenantObj, Span span, Handler<AsyncResult<OperationResult<Id>>> resultHandler);
+    void createTenant(Optional<String> tenantId, JsonObject tenantObj, Span span, Handler<AsyncResult<OperationResult<Id>>> resultHandler);
```

Comment:
`TenantManagementService` is using `JsonObject` instead of `Tenant` value object